src/cascadia/TerminalApp/App.cpp

```diff
@@ -231,6 +233,18 @@ namespace winrt::TerminalApp::implementation
                 profileMenuItem.Icon(_GetIconFromProfile(profile));
             }
 
+            if (profile.GetGuid() == defaultProfileGuid)
+            {
+                // Contrast the default profile with others in font weight.
+
+                // The "bold" and "normal" font weights are 700 and 400 respectively,
+                // so we add 300 to get bold from normal.
+                // If the default font weight is higher, so is the accented one (but no higher than 999).
+                const uint16_t defaultFontWeight = profileMenuItem.FontWeight().Weight;
+                const uint16_t accentedFontWeight = gsl::narrow<uint16_t>(defaultFontWeight + 300u);

```

Comment:
I like this because it'll have ane ver increasing boldness, but... should we just use `winrt::Windows::UI::Text::FontWeights::Bold()`?

Reply:
I actually have no idea whether there can exist situations where the default font weight of the flyout text is not `FontWeights::Normal()`.  If there can be none then it's much clearer to just use `FontWeights::Bold()` instead indeed...

Reply:
Let's move to `Bold()` for now until we can understand the context in which people will be using our UI

Reply:
:smile: thanks